templates/pages/home.html

```diff
@@ -27,14 +28,17 @@
 <div class="main full">
     {{#if products.featured}}
         {{> components/products/featured products=products.featured columns=theme_settings.homepage_featured_products_column_count}}
+        {{{region name="page_content"}}}
     {{/if}}
 
     {{#if products.top_sellers}}
         {{> components/products/top products=products.top_sellers columns=theme_settings.homepage_top_products_column_count}}
+        {{{region name="page_content"}}}
```

Comment:
Could be better named to indicate its relation to Top Selling Products

Reply:
I am not sure if we should have regions inside very specific if conditions like these. If we do then we need to be very specific with the naming as Nathan pointed out above.